### ISSUE TITLE
bgithub1/web-clnt-access/1.0 build issue

### ISSUE THREAD

**Issue by bgithub1** [jitpack/jitpack.io#73]
Log https://jitpack.io/com/github/bgithub1/web-clnt-access/1.0/build.log

Hi Andrejs.  I think there is a problem with my old web-clnt-access build still being stuck somewhere on jitpack, so that the newer version that I put on https://github.com/bgithub1/web-clnt-access is not being accessed by jitpack.  When I delete a version of https://github.com/bgithub1/web-clnt-access by hitting the [x] next to it's status, the get button does not seem to actually get a new version.  It doesn't seem to do anything.  Then, when I delete the com/github/bgithub1/web-clnt-access folder in my .m2/repository, and I execute mvn compile on the project that references web-clnt-access, all of the same errors occur again.  jitpack seems to be referencing an old version of web-clnt-access that it cached.  

Is that possible?

**Comment by jitpack-io** [comment]
Hi,
Version 1.0 of web-clnt-access  is quite old, it doesn't have the jitpack.io repository in it.
Creating a new release of web-clnt-access should help

**Comment by bgithub1** [comment]
Huh, old? (Andrejs - this can't be you that I'm talking to)

I just made it public, and created release 1.0 about an hour ago.  And, it
does have the jitpack.io repository in it. (which I use in another project
and works).

If you are seeing a version of Version 1.0 that doesn't have the jitpack.io
repository tag in it, then you are not going out to
https://com.github/bgithub1/web-clnt-access to fetch a the latest copy of
that version after, I have deleted it on your website.

-Bill Perlman
